setup.cfg

```diff
@@ -8,7 +8,8 @@ install_requires =
     boto3==1.26.45
     pandas==1.5.2
     pydantic==1.10.4
-
+    jinja2==3.1.2
+    PyYAML==6.0
 [options.extras_require]
```

Comment:
Somewhere in here I think we need to include something like the following so the `workflow_template.yaml` file is installed with the package:

```
[options.package_data]
enclave_mgmt =
    *.yaml
```

Otherwise, something like the following will fail because it will look for and not find `/usr/local/lib/python3.10/site-packages/enclave_mgmt/workflow_template.yaml`:

```bash
$ docker build . -t raise-enclave
$ docker run --rm raise-enclave create-workflow a b c 
```